### ISSUE TITLE
Add mlr3extralearners and mlr3proba to Suggests

### ISSUE THREAD

**Issue by sebffischer** [mlr-org/mlr3verse#40]
This depends on whether Additional_repositories allow github dependencies.

**Comment by bblodfon** [comment]
`mlr3proba` is already in `Suggests` (I will update when `mlr3survival` is ready). What about `mlr3extralearners` though, do we want it installed from `Suggests`? @be-marc

**Comment by be-marc** [comment]
I don't think that changes anything. If `mlr3extralearners` is in `Additional Repositories`, `install.packages()` and `pak()` will not install `mlr3extalearners` when you install `mlr3verse`.

**Comment by bblodfon** [comment]
ok, lets include it then!

**Comment by be-marc** [comment]
Done
